### ISSUE TITLE
scrapping of bosh api metrics server fails with the latest bosh version

### ISSUE THREAD

**Issue by JCL38-ORANGE** [orange-cloudfoundry/paas-templates#1702]
### Expected behavior
* As a paas-templates operator
* In order to be confident with my platform
* I need all prometheus targets scrapping succeed

### Observed behavior
When we attempt to get metrics locally on bosh director VM, a ruby stack trace is raised (see uptream bug for the complete stack trace)
```
curl -X GET http://localhost:25556/metrics
```
Consequently, scrapping via prometheus fails.

### Tips
Metrics can be scrapped locally by
```
http://localhost:9092/metrics
```

API Metrics can be scrapped locally by
```
curl http://localhost:25556/metrics
```

### Affected releases
v54.0.2

### Upstream bug
https://github.com/cloudfoundry/bosh/issues/2394

**Comment by gberche-orange** [comment]
Also reproduced in v52 https://github.com/orange-cloudfoundry/paas-templates/issues/1412 see reported upstream bug https://github.com/cloudfoundry/bosh/issues/2332

**Comment by JCL38-ORANGE** [comment]
Workaround implemented in feature-fix-54.0.2 (remove /var/vcap/store/director/metrics bin files before starting)
